test/test-https.js

```diff
@@ -51,9 +52,19 @@ tape('setup', function (t) {
 });
 
 tape('make request to https server', function (t) {
+	got('https://google.com', {
+		strictSSL: true
+	}, function (err, data) {
+		t.error(err);
+		t.ok(data);
+		t.end();
+	});
+});
+
+tape('make request to https server with ca', function (t) {
 	got(s.url, {
 		strictSSL: true,
-		ca: caRootCert,
+		agent: new https.Agent({ca: caRootCert}),
```

Comment:
I'm only worried about this - `ca` option should be passed to `tls.connect`, but it does not.